apps/home/templates/apps_home/blocks/faq_block.html

```diff
@@ -6,7 +6,7 @@
 <section class="modul-servicepanel fullwidth ds-block {% if self.background_color %}ds-block--{{ self.background_color }}{% else %}ds-block--body-bg{% endif %}">
     <div class="servicepanel__main">
         {% if self.faq_title %}
-        <h2 class="faq-ds-block__title faq-ds-block__title--dark">
+        <h2 class="title">
```

Comment:
I guess the styling was already changed and the classes are removed from the css as well?

Reply:
styling and classes were removed in the story pull request